### ISSUE TITLE
Risks

### ISSUE THREAD

**Issue by Charles-Hoot** [Dixith1196/THE-HUNT#10]
I don't see how jailbreak on android/iOS is a relevant risk for this app.  There are security risks associated with this app, but 

the big risks I see (aside from location hiding) are: Specifying a location, loss of data service, and creating/managing teams.

**Comment by Ravichanderreddy-goli** [comment]
Yeah, actually jailbreak is not that an issue for our app. Creating and managing the teams, loss of data service are bigger risks for our app. Thank you for guiding us.
